packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue

```diff
@@ -22,6 +23,9 @@ export default {
     Simplebar,
     SfButton,
   },
+  directives: {
```

Comment:
we need it here? SfButton has v-focus if I think good

Reply:
yes, this is just an import which we dont need here :) thanks